src/AI/WallyAI.java

```diff
@@ -400,7 +399,7 @@ else if( (null == resident) && myCo.unitProductionByTerrain.containsKey(gameMap.
         boolean foundAction = false;
 
         // Find the possible destinations.
-        ArrayList<XYCoord> destinations = Utils.findPossibleDestinations(unit, gameMap, true);
+        ArrayList<XYCoord> destinations = Utils.findPossibleDestinations(unit, gameMap, false);
```

Comment:
This `false` used to be `true`, FYI.

Reply:
Only ATTACK and CAPTURE are considered in that block, though. I guess I should do the temp var thing anyway.